guilford_sent_component.yaml

```diff
@@ -0,0 +1,597 @@
+
+  sentenceterm:
+    fqn: "ol.sentenceterm"
+    entitySetName: "GuilfordSentenceTerms_DPS"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["OFFENDER NC DOC ID NUMBER", "SENTENCE EFFECTIVE(BEGIN) DATE"] #just these two columns in order to overlap with sent_computation table
+      ol.minimumvalue:
+        type: "ol.minimumvalue"
+        column: "MINIMUM SENTENCE LENGTH"
+        transforms:
+        - !<transforms.ParseDoubleTransform> {}
+      ol.maximumvalue:
+        type: "ol.maximumvalue"
+        column: "MAXIMUM SENTENCE LENGTH"
+        transforms:
+        - !<transforms.ParseDoubleTransform> {}
+      ol.datetimestart:
+        type: "ol.datetimestart"
+        column: "SENTENCE EFFECTIVE(BEGIN) DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S", "yyyy-MM-dd HH:mm:ss.SS", "yyyy-MM-dd HH:mm:ss.SSS"]
+          timezone: "America/New_York"
+    name: "sentenceterm"
+
+  jailtimemodifications:
+    fqn: "ol.jailtimemodifications"
+    entitySetName: "BerksJailTimeCredit"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "CREDITS FOR JAIL DAYS SERVED"
+      ol.value:
+        type: "ol.value"
+        column: "CREDITS FOR JAIL DAYS SERVED"
+        transforms:
+        - !<transforms.ParseDoubleTransform> {}
+      ol.units:
+        type: "ol.units"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "Days"
+    name: "jailtimemodifications"
+
+  icccreditdays:
+    fqn: "ol.jailtimemodifications"
+    entitySetName: "BerksICCCredit"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "ICC JAIL CREDITS (IN DAYS)"
+      ol.value:
+        type: "ol.value"
+        column: "ICC JAIL CREDITS (IN DAYS)"
+        transforms:
+        - !<transforms.ParseDoubleTransform> {}
+      ol.units:
+        type: "ol.units"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "Days"
+      ol.type:
+        type: "ol.type"
+        column: "ICC JAIL CREDITS (IN DAYS)"
+    name: "icccreditdays"
+
+  disposition:
+    fqn: "ol.disposition"
+    entitySetName: "GuilfordDispositions"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "COMPONENT DISPOSITION CODE"
+      criminaljustice.disposition:
+        type: "criminaljustice.disposition"
+        column: "COMPONENT DISPOSITION CODE"
+    name: "disposition"
+
+  jailstay:
+    fqn: "ol.jailstay"
+    entitySetName: "GuilfordJailstays_DPS"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["OFFENDER NC DOC ID NUMBER", "SERVING MIN OR MAX TERM CODE"]
+      ol.minmax:
+        type: "ol.minmax"
+        column: "SERVING MIN OR MAX TERM CODE"
+    name: "jailstay"
+
+  offense:
```

Comment:
Did we decide that offense was unnecessary, being redundant with "charge"?

Reply:
We did but I re-added it since offense start and end datetimes can only go here.